devops/git-flow/README.md

```diff
@@ -0,0 +1,127 @@
+# Git Flow
+
+In this standard, we suggest you multiple implementations of Git Flow. Each flow is different. Some is better when you're developing, some is better when your project was already go live. This standard will help you pick the right one for your project.
+
+Before we start, one thing that's constant is we will use only _one branch_ as a standard.
+
+## TL;DR
+
```

Comment:
in TL;DR, I think we should add short description on each git flow. For example

* **Centralized Workflow**: have only master branch and all changes are committed into master.
* **Feature Branch**: have master and feature branch.
* **Trunk-base Development**: it is like feature branch but feature branch is very short-lived and will merge to master as fast as possible. Project should implement feature flags turn on/off feature quickly if bugs occurred.

Reply:
Updated krubbb